### ISSUE TITLE
php@7.3-mongodb-1.7.4_1 needs rebuild for icu

### ISSUE THREAD

**Issue by glensc** [kabel/homebrew-pecl#7]
```
➔ brew install php@7.3-mongodb
==> Installing php@7.3-mongodb from kabel/pecl
==> Downloading https://dl.bintray.com/kabel/bottles-pecl/php%407.3-mongodb-1.7.4_1.catalina.bottle.tar.gz
Already downloaded: /Users/glen/Library/Caches/Homebrew/downloads/5aded31391d50267cf40429f54652a48bd34b3a0ec41607bcf1be4409fe74bfb--php@7.3-mongodb-1.7.4_1.catalina.bottle.tar.gz
==> Pouring php@7.3-mongodb-1.7.4_1.catalina.bottle.tar.gz
==> Caveats
php@7.3-mongodb is keg-only, which means it was not symlinked into /usr/local,
because this is an alternate version of another formula.

For compilers to find php@7.3-mongodb you may need to set:
  export LDFLAGS="-L/usr/local/opt/php@7.3-mongodb/lib"
  export CPPFLAGS="-I/usr/local/opt/php@7.3-mongodb/include"

==> Summary
🍺  /usr/local/Cellar/php@7.3-mongodb/1.7.4_1: 7 files, 1.2MB

➔ php --version
PHP Warning:  PHP Startup: Unable to load dynamic library '/usr/local/opt/php@7.3-mongodb/lib/php/20180731/mongodb.so' (tried: /usr/local/opt/php@7.3-mongodb/lib/php/20180731/mongodb.so (dlopen(/usr/local/opt/php@7.3-mongodb/lib/php/20180731/mongodb.so, 9): Library not loaded: /usr/local/opt/icu4c/lib/libicuuc.66.dylib
  Referenced from: /usr/local/opt/php@7.3-mongodb/lib/php/20180731/mongodb.so
  Reason: image not found), /usr/local/lib/php/pecl/20180731//usr/local/opt/php@7.3-mongodb/lib/php/20180731/mongodb.so.so (dlopen(/usr/local/lib/php/pecl/20180731//usr/local/opt/php@7.3-mongodb/lib/php/20180731/mongodb.so.so, 9): image not found)) in Unknown on line 0

➔ l /usr/local/opt/php@7.3-mongodb/lib/php/20180731/mongodb.so
-r--r--r-- 1 glen staff 1.2M juuni  5 11:40 /usr/local/opt/php@7.3-mongodb/lib/php/20180731/mongodb.so
```

**Comment by glensc** [comment]
@kabel I waited couple of days reporting this, hoping you figure this yourself, but seems no. so hence the report here.

**Comment by glensc** [comment]
until the new bottle is available, can rebuild from source locally:

```
➔ brew reinstall php@7.3-mongodb -s
```

**Comment by kabel** [comment]
Fixed at 47b8a96c6f222d4147747b5e74a616e116923ffb. Feel free to drop a PR to kick off a new bottle build. I'm not always up to speed on the upstream changes these days.
